include/odp/api/spec/packet_io_stats.h

```diff
@@ -26,42 +26,23 @@ extern "C" {
  */
 
 /**
- * Packet IO statistics
- *
- * Packet IO statistics counters follow RFCs for Management Information Base
- * (MIB)for use with network management protocols in the Internet community:
- * https://tools.ietf.org/html/rfc3635
- * https://tools.ietf.org/html/rfc2863
- * https://tools.ietf.org/html/rfc2819
+ * Packet IO statistics counters
  */
 typedef struct odp_pktio_stats_t {
-	/**
-	 * The number of octets in valid MAC frames received on this interface,
-	 * including the MAC header and FCS. See ifHCInOctets counter
-	 * description in RFC 3635 for details.
-	 */
+	/** Number of octets in successfully received packets (includes MAC
+	 *  headers and FCSs). */
 	uint64_t in_octets;
 
-	/**
-	 * The number of packets, delivered by this sub-layer to a higher
-	 * (sub-)layer, which were not addressed to a multicast or broadcast
-	 * address at this sub-layer. See ifHCInUcastPkts in RFC 2863, RFC 3635.
-	 */
+	/** Number of successfully received packets with a unicast destination
+	 *  MAC address. */
 	uint64_t in_ucast_pkts;
 
-	/**
-	 * The number of inbound packets which were chosen to be discarded
-	 * even though no errors had been detected to prevent their being
-	 * deliverable to a higher-layer protocol.  One possible reason for
-	 * discarding such a packet could be to free up buffer space.
-	 * See ifInDiscards in RFC 2863.
-	 */
+	/** Number of inbound packets which were discarded, even though no
+	 *  errors had been detected. This can be caused by a lack of free
+	 *  resources (e.g. buffers). */
 	uint64_t in_discards;
 
-	/**
-	 * The sum for this interface of AlignmentErrors, FCSErrors,
-	 * FrameTooLongs, InternalMacReceiveErrors. See ifInErrors in RFC 3635.
-	 */
+	/** Number of inbound packets with errors. */
```

Comment:
Clarification may be needed: Depending on packet input configuration, packets with errors may be dropped or not.

Reply:
Fixed in V2